### PR TITLE
Support vertex color shader in jassimp

### PR DESCRIPTION
**Pull request by NolaDonato**
If vertex colors are found in a mesh, Jassimp will now use GVRVertexColorShader instead of GVRPhongShader. The vertex color shader has been rewritten to allow lighting if normals are also present in the mesh.

GearVRF DCO signed off by: Nola Donato nola.donato@samsung.com

This PR has not been tested yet. It is for Mitch to review.

### CODE REVIEW

File: GVRf/Framework/framework/src/main/java/org/gearvrf/shaders/GVRTextureShader.java [hunk 1]
```diff
@@ -29,10 +29,6 @@
 
 import org.gearvrf.R;
 
-/**
```
Comment (thomasflynn) [inline GVRf/Framework/framework/src/main/java/org/gearvrf/shaders/GVRTextureShader.java]: why was the javadoc removed for a public class?

File: GVRf/Framework/framework/src/main/res/raw/vcolor_vertex.vsh [hunk 2]
```diff
@@ -1,31 +1,113 @@
+#e
```
Comment (thomasflynn) [inline GVRf/Framework/framework/src/main/res/raw/vcolor_vertex.vsh]: is this "#e" a typo?

**Comment by thomasflynn** [comment]
@mwitchwilliams you marked this as 'needs work'.  can you leave a comment as to why?

File: GVRf/Framework/framework/src/main/res/raw/vcolor_vertex.vsh (hunk 2 above)
Comment (mwitchwilliams) [reply GVRf/Framework/framework/src/main/res/raw/vcolor_vertex.vsh]: This PR uses a different V & F shaders than I used in PLYdemo (https://github.com/gearvrf/GearVRf-Demos/pull/647).  PLYdemo uses the same V & F shaders used by line drawing.

Comment (mwitchwilliams) [reply GVRf/Framework/framework/src/main/res/raw/vcolor_vertex.vsh]: I removed the "#e" and it seemed to make no difference.  I still didn't see the PLY object.
I even re-built Framework and sync'd gradle again, and same results.

**Comment by liaxim** [comment]
Should we close?

**Pull request closed without merging**